client/src/Components/GoogleMap/GoogleMap.js

```diff
@@ -1,50 +1,77 @@
 import React, { Component } from 'react';
-import { Map, GoogleApiWrapper } from 'google-maps-react';
+import { Map, GoogleApiWrapper, Marker, Polyline } from 'google-maps-react';
 import { Header, Icon, Segment } from 'semantic-ui-react';
 
 class SimpleMap extends Component {
-  // constructor(props) {
-  //   super(props);
+  render() {
+    // trip end and start location
+    const tripPoints = this.props.trips
+      .map(trip => {
+        const tripStart = trip.startLocation;
+        const tripEnd = trip.endLocation;
 
-  //   // this.state = {
-  //   //   stores: [{ lat: 47.49855629475769, lng: -122.14184416996333 },
-  //   //   { latitude: 47.359423, longitude: -122.021071 },
-  //   //   { latitude: 47.2052192687988, longitude: -121.988426208496 },
-  //   //   { latitude: 47.6307081, longitude: -122.1434325 },
-  //   //   { latitude: 47.3084488, longitude: -122.2140121 },
-  //   //   { latitude: 47.5524695, longitude: -122.0425407 }]
-  //   // }
-  // }
+        return {
+          locations: [tripStart?.geoPoint, tripEnd?.geoPoint],
+          numberOfEvents: parseFloat(trip.numberOfEvents) / trip.distance,
+        };
+      })
+      .filter(tripData => {
+        const [startLocation, endLocation] = tripData.locations;
+        return startLocation && endLocation;
+      });
 
-  // displayMarkers = () => {
-  //   //TODO;
-  //   // return parkData.features.map(park => {
-  //   //   return (
-  //   //     <Marker
-  //   //       key={park.properties.PARK_ID}
-  //   //       position={{
-  //   //         lat: park.geometry.coordinates[1],
-  //   //         lng: park.geometry.coordinates[0]
-  //   //       }}
-  //   //       onClick={() => console.log("You clicked me!")}
-  //   //     />
-  //   //   );
-  //   // });
-  // };
+    // centers the map to last location
```

Comment:
not really a correct name. `tripsWithEndLocations` would be a more correct name

Reply:
Also this seems redundant as we already have tripPoints. You can use that for the initial centered point